index.html

```diff
@@ -4,23 +4,70 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Static Template</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;700;900&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Indie+Flower&family=Roboto:wght@100;700;900&display=swap"
-      rel="stylesheet"
-    />
+    <!--CSS Styles-->
+    <link href="./styles/master.css" rel="stylesheet" />
+    <title>Magazine</title>
   </head>
+
   <body>
-    <h1 class="header" id="header1">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h2 class="header" id="header2">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h3 class="header" id="header3">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h4 class="header" id="header4">Hola <b>#BootcampMex</b> 2021!</h1>
+    <div class="conteiner"> <!--Body of magazine--> 
+      <form class="magForm"> <!--Space from magazine border to center--> 
+        <br />
+        <!--Magazine Title and description-->
```

Comment:
casi siempre no es buena practica dejar comentarios en el codigo. ya que esto se puede cubrir escribiendo un html semantico y un buen naming de variables/clases/functions etc

Reply:
usando el Heading 1 y un span/paragraph me dice lo mismo que tu comentario 
`<!--Magazine Title and description-->`